recipe/run_test.sh

```diff
@@ -39,7 +39,7 @@ python -m numba.tests.test_runtests
 
 if [[ "$archstr" == 'aarch64' ]]; then
 	echo 'Running only a slice of tests'
-	$SEGVCATCH python -m numba.runtests -b -j --random='0.15' --exclude-tags='long_running' -m $TEST_NPROCS -- numba.tests
+	$SEGVCATCH python -m numba.runtests -b -j --random='0.10' --exclude-tags='long_running' -m $TEST_NPROCS -- numba.tests
```

Comment:
```suggestion
	$SEGVCATCH python -m numba.runtests -b -j --random='0.15' --exclude-tags='long_running' -m $TEST_NPROCS -- numba.tests
```
suggest reinstating this now the time out on that one particular failing test group is extended.